handbook/product/licensing.md

```diff
@@ -0,0 +1,28 @@
+# Licensing
```

Comment:
I think it would be useful to link to https://about.sourcegraph.com/handbook/ce/enterprise-vs-oss somewhere on this page. Perhaps at the bottom of this top section of the page? Or down in the "Summary" section? But seems like a useful link to have here!

Reply:
Agreed! Added to the summary.